app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/SessionDetailFragment.java

```diff
@@ -85,6 +79,17 @@ private void initTheme() {
                     new ActivityManager.TaskDescription(viewModel.getSessionTitle(), null,
                             ContextCompat.getColor(activity, viewModel.getSessionVividColorResId()));
             activity.setTaskDescription(taskDescription);
+
+            // Change status bar scrim color
+            Window window = activity.getWindow();
```

Comment:
Thank you for your comment!
Certainly this code was unnecessary. Delete unnecessary code and push it.